source/vstd/raw_ptr.rs

```diff
@@ -474,7 +529,7 @@ pub broadcast group group_raw_ptr_axioms {
     ptrs_mut_eq,
 }
 
-// Exposing provenance
+/// Mechanism for exposing provenance.
```

Comment:
Could be more explicit, "Tracked object that indicates a given provenance has been exposed."

Reply:
done